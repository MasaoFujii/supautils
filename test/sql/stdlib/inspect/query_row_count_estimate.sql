select supa.query_row_count_estimate($$
    select *
    from generate_series(1, 100);
$$::text);
