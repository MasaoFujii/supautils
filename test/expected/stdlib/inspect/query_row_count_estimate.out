select supa.query_row_count_estimate($$
    select *
    from generate_series(1, 100);
$$::text);
 query_row_count_estimate 
--------------------------
                      100
(1 row)

